hexa/pipelines/models.py

```diff
@@ -465,17 +474,8 @@ def last_version(self) -> "PipelineVersion":
         return self.versions.first()
 
     @property
-    def is_new_template_version_available(self) -> bool:
-        if (
-            self.source_template
-            and hasattr(self.last_version, "source_template_version")
-            and self.last_version.source_template_version
-        ):
-            return (
-                self.source_template.last_version.created_at
-                > self.last_version.source_template_version.created_at
-            )
-        return False
+    def new_template_versions(self):
+        return self.source_template.new_versions(self) if self.source_template else []
```

Comment:
Looking at this sole line, My first idea would be to implement this on the query set of `PipelineTemplateVersion` as `PipelineTemplateVersion.objects.get_updates_for(pipeline:Pipeline)`

Reply:
And then I would maybe re-evaluate the need for this property.

Reply:
I'm ok with this anyway.